server/db/migrations/13_recreate_access_tokens.rb

```diff
@@ -0,0 +1,7 @@
+class RecreateAccessTokens < Mongodb::Migration
+  def self.up
+    AccessToken.collection.drop
```

Comment:
Is there any sane way to migrate old tokens to new ones? If yes, we should implement that in a separate PR. This is ok implementation for this PR :)